apps/test/netsim/NetSimRouterNode.js

```diff
@@ -30,6 +32,54 @@ describe("NetSimRouterNode", function () {
     testShard = fakeShard();
   });
 
+  describe("default row structure", function () {
+    var row;
+
+    beforeEach(function () {
+      var router = new NetSimRouterNode(testShard);
+      row = router.buildRow_();
+    });
+
+    it ("dnsMode (default DnsMode.NONE)", function () {
+      assertOwnProperty(row, 'dnsMode');
+      assertEqual(row.dnsMode, DnsMode.NONE);
+    });
+
+    it ("dnsNodeID (default undefined)", function () {
+      assertOwnProperty(row, 'dnsNodeID');
+      assertEqual(row.dnsNodeID, undefined);
+    });
+
+    it ("bandwidth (default 'Infinity')", function () {
+      assertOwnProperty(row, 'bandwidth');
+      assertEqual(row.bandwidth, 'Infinity');
+    });
```

Comment:
It's true, there's not actually a _need_ to test these in isolation.  Maybe I just wanted to see the field names in my test output :smiley: I'll do a pass and clean this pattern up across my entity tests.